client/src/containers/pages/Home/tests/__snapshots__/Home.test.tsx.snap

```diff
@@ -0,0 +1,221 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`containers/pages/Home renders Home correctly & changes Title of page: Home page rendered 1`] = `
```

Comment:
This snapshot is too large.

- Make it smaller please.
- Check why the lint-snap test did not flag it and fix it

Reply:
Please see https://kentcdodds.com/blog/effective-snapshot-testing/

Reply:
created an issue here to track this https://github.com/OpenSRP/web/issues/60